apps/api/src/report/report.util.service.ts

```diff
@@ -229,22 +231,54 @@ export class ReportUtilService {
   */
   licensingStageApplicantsQuery(from: string, to: string) {
     return `
-      --col01 - 'Applied to NNAS' status: 4, 5, 6
-      --col02 - 'Applied to BCCNM' status: 7
-      --col03 - 'Completed English Language Requirement' status: 8
-      --col04 - 'Referred to NCAS' status: 9,10,11
-      --col05 - 'Completed NCAS (BCCNM Assessment)' status: 12
-      --col06 - 'Completed Additional Education' status: 13, 14
-      --col07 - 'Referred to NCLEX' status: 15,17
-      --col08 - 'NCLEX - Passed' status: 18
-      --col09 - 'BCCNM Licensed - Full License' status: 19
-      --col10 - 'Granted full licensure' status: 19
-      --col11 - 'Granted provisional licensure' status: 16
+      --col01 - 'Applied to NNAS' status: 4, 5, 6 e68f902c-8440-4a9c-a05f-2765301de800
```

Comment:
Even though we provide these comments for better understanding, it is hard to grasp the meaning of IDs in the query.
How about passing a map with keys as status and values as id to each report util function?